src/ui/Templates/HtmlTemplate.ts

```diff
@@ -19,7 +19,10 @@ export class HtmlTemplate extends Template {
 
   toHtmlElement(): HTMLElement {
     let script = this.templateFromAScriptTag.toHtmlElement();
-    script.setAttribute('type', _.first(HtmlTemplate.mimeTypes));
+    // We don't set the type attribute for 2 reasons:
```

Comment:
Instead of relying on a comment to not break this, could we have a UT for this explicitly ?

Any mechanism we could have to remove the `type` attribute if we detect that it's set ?

What happens exactly when that value is set and the evil locker service is activated ?

Does it throw ? I'd like more information about what we are trying to prevent.